generate_solution_visionos.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# *** Command Line Arguments *** 
+# DLLOnly - Generates a solution with only DLL-related build configurations
+python3 -m pip install -r teamcity/requirements.txt
+
+git config core.hooksPath .githooks
+git config commit.template .githooks/commit-template.txt
+
+XCODE_VERSION=$(softwareupdate --history | awk '/Command Line Tools for Xcode/ {print $6}' | tail -1)
+
+if [ ! -f /modules/premake/README.md ]
+then
+	git submodule update --recursive
+fi
+
+
+if [ ! -f /modules/premake/bin/release/premake5 ]
+then
+	MIN_XCODE_VERSION="14.2"
```

Comment:
The first public release of visionOS was [introduced](https://xcodereleases.com/) in Xcode 15.2, it's very likely this won't work as expected in previous versions.

Reply:
I can dig that

Reply:
Fixed in latest.